### PR TITLE
Added Loft Labs to the lists

### LINKED ISSUE #772: Add Loft Labs to the Hacktoberfest Swag List
Loft Labs is participating in the Hacktoberfest 2023!

Contributions to their any of the Loft Labs open source projects, vCluster and devpod projects and get prizes for every merged PR.

ref [Loft Labs on twitter](https://twitter.com/loft_sh/status/171033134552680071)

loft prizes

This issue is open for anyone who wants to contribute to this repo.

Please read the [contributing guideline](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) to understand the formatting and the information expected from the announcement for your PR to be successfully merged.

### PR DESCRIPTION
**Pull request by ikunal-04**
Fixes #772 
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [✅] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [✅] I'm not adding a company from the blocklist
3. [✅] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

File: docs/list.md [hunk 1]
```diff
@@ -299,6 +299,16 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
 - **Notes**: Read more at [JavaScript Today](https://blog.javascripttoday.com/blog/hacktoberfest-2023/).
 
+### L
+
+#### **Loft Labs**
+
+- **Swag**: Loft exclusive prizes.
+- **Requirements**:
+  - Contribute to vcluster and devpod project of Loft Labs.
+- **How to sign up**: Join the [loft labs slack channel](https://slack.loft.sh/)
```
Comment (vimode) [inline docs/list.md:309]: I don't see this as a requirement in the tweet. 
Was this updated somewhere? Can you add that link too?

Please update the link twitter link below with the actual tweet link or the updated announcement link.

Comment (ikunal-04) [reply docs/list.md:309]: Yes they have mentioned in one of the tweet that only these 2 projects would be counted for exclusive prizes [tweet link](https://twitter.com/loft_sh/status/1710331345526800719).
Working on all the other suggestions you have made 👍

Comment (vimode) [reply docs/list.md:309]: I am sorry, I meant to ask about the sign up part. The requirement to join their slack channel.

Comment (ikunal-04) [reply docs/list.md:309]: Oh yess it's a mistake there. There is no any special sign up is known for loft lab. I'm sorry updating this 👍. And add on I have updated the specific tweet link and added the issues part mentioning issues only for devpod and vcluster.

**Comment by crweiner** [comment]
Howdy @ikunal-04! Thank you for adding Loft Labs to the Hacktoberfest Swag List. And thank you too @vimode for the review! I made some small tweaks, so now we're good to get Loft Labs on the List. ✅ Have a great Hacktoberfest!

thank you

**Pull request merged**